service/src/java/org/apache/hive/service/cli/thrift/ThriftHttpCLIService.java

```diff
@@ -24,10 +24,13 @@
 import java.util.concurrent.SynchronousQueue;
 import java.util.concurrent.ThreadPoolExecutor;
 import java.util.concurrent.TimeUnit;
+import java.util.Set;
 
 import javax.net.ssl.KeyManagerFactory;
 import javax.ws.rs.HttpMethod;
 
+import com.google.common.base.Splitter;
```

Comment:
Does this import add new compile time dependencies for HS2 service module? I am just trying to make sure that this binary is already being used else where in this code, so we are not adding a new runtime dependency for this code.

Reply:
It is not new, it has been used by HttpServer.java